frontend/src/__tests__/api.test.js

```diff
@@ -253,9 +387,41 @@ function positionsTests(api = { apiGET, apiPOST }) {
         checkPropTypes(errorPropTypes, resp1);
     });
 
-    it.todo(
-        "succeed when creating two positions with the same code but for different sessions"
-    );
+    it("succeed when creating two positions with the same code but for different sessions", async () => {
+        const newSessionData = {
+            start_date: "2019/09/09",
+            end_date: "2019/12/31",
+            // add a random string to the session name so we don't accidentally collide with another
+            // session's name
+            name: "Newly Created Sessions (" + Math.random() + ")",
+            rate1: 56.54
+        };
+        const newPositionData2 = {
+            position_code: "MAT135F",
+            position_title: "Calculus I",
+            est_hours_per_assignment: 70,
+            est_start_date: "2019/09/09",
+            est_end_date: "2019/12/31",
+            position_type: "Standard"
+        };
+        // create a new session to add a template to
+        const resp1 = await apiPOST("/sessions", newSessionData);
+        expect(resp1).toMatchObject({ status: "success" });
+        const sessionId = resp1.payload.id;
+
+        // create a new position of the same code associated with new session
+        const resp2 = await apiPOST(
+            `/sessions/${sessionId}/positions`,
+            newPositionData2
+        );
+        expect(resp2).toMatchObject({ status: "success" });
+        // make sure we get back what we put in
+        expect(resp2.payload).toMatchObject(newPositionData2);
+
+        // make sure the position is created
+        const resp3 = await apiGET(`/sessions/${sessionId}/positions`);
+        expect(resp3.payload).toContainObject(resp2.payload);
```

Comment:
Do we also have a test that ensures that creating two positions with the same code in the same session indeed fails?